wasm_bindgen/private/wasm_bindgen.bzl

```diff
@@ -96,7 +96,7 @@ def _rust_wasm_bindgen_impl(ctx):
 
     return [
         DefaultInfo(
-            files = depset(transitive = [info.js, info.ts]),
+            files = depset(transitive = [info.js, info.ts, depset([info.wasm])]),
```

Comment:
Why not this?
```suggestion
            files = depset([info.wasm], transitive = [info.js, info.ts]),
```

Reply:
I wasn't aware this was an option! Let's go with this instead.